modules/channel-web/src/backend/api.ts

```diff
@@ -205,10 +212,14 @@ export default async (bp: typeof sdk, db: Database) => {
         user.result
       )
 
-      return res.sendStatus(200)
+      res.sendStatus(200)
     })
   )
 
+  router.post('/messages', (req, res) => {
```

Comment:
just in case one uses this api

Reply:
You send a 301 Moved Permanently, but doesn't redirect to any URL is that normal or am I missing something?

Reply:
this was for convenience, but you're right it's confusing. I simply got rid of it